### ISSUE TITLE
Configure layout title form does not return

### ISSUE THREAD

**Issue by alanmels** [backdrop/backdrop-issues#4707]
**Description of the bug**

If you go to `admin/structure/layouts/manage/default/edit-title/editor/title` and click on any option, then the page doesn't return. and it's not clear if the new value is saved until you manually refresh the page. I believe it happens because the submit button is ajaxified unlike other submit button in Layouts configuration pages.

**Steps To Reproduce**

To reproduce the behavior:

1. Go to Structure > Layouts > (choose any layout) > Configure page title
2. On the opened page click on any option and Save Configuration 

**Expected behavior**

I believe saving the new configuration for title should return to `admin/structure/layouts/manage/home` page.

**Comment by BWPanda** [comment]
Yeah, this is because you're opening a form on a separate page instead of in a modal like it's designed for.

I wonder, does it work properly if JS is disabled?

**Comment by alanmels** [comment]
@BWPanda, please see the problem persists even if you do not open the link directly, but use the admin bar instead:

![title](https://user-images.githubusercontent.com/56604405/96354485-793a4c00-1094-11eb-962e-c31e22d11f11.gif)

So the modal is not kicking in. However, I believe the issue must be properly fixed even the modal was there (for cases, when someone tries to open that page directly).

**Comment by BWPanda** [comment]
> the problem persists even if you do not open the link directly, but use the admin bar instead

Ha! I'd never even noticed that was possible :smile: 

Confirmed this bug. Definitely that page should work properly since it's accessible without a module popup.

**Comment by stpaultim** [comment]
I was looking at issue #3983 and learned of this bug. 

While I like some of the ideas in #3983, they don't seem as urgent as this bug. We have a menu link to a page this is simply broken and apparently has been for a long time. 

A simple short term solution would be to remove this link from the admin menu if we can't get it to work. There are other ways to edit the title. Given that I don't remember ever editing the title of a layout this way, it may not be too urgent. But, clearly some people are finding this bug.
